HoloPipelines/jobs/jobs_state.py

```diff
@@ -5,17 +5,24 @@
 import logging
 import threading
 import time
-from datetime import datetime
 from enum import Enum
+from typing import List
 
 from config import (
     KEEP_ALL_FILES,
     KEEP_ALL_LOG_FILES,
     GARBAGE_COLLECTION_INTERVAL_SECS,
     GARBAGE_COLLECTION_INACTIVE_JOB_TTL_SECS,
 )
-from jobs import jobs_state_dict
-from jobs.jobs_io import remove_directory_for_job
+from jobs.jobs_io import (
+    get_all_job_subdirectory_names,
+    read_state_file_for_job,
+    remove_temporary_data_for_job,
+    remove_log_file_for_job,
+    move_job_to_finished_jobs_directory,
+    write_state_file_for_job,
+    state_file_for_job_exists,
+)
 
 JobState = Enum(
```

Comment:
`JobState` does not feel like a proper python variable naming convention for global variables. The naming convention should be similar to python functions.

https://www.python.org/dev/peps/pep-0008/#global-variable-names

Reply:
Hm but it is an Enum right? See for instance how they use it in the docs: https://docs.python.org/3/library/enum.html#functional-api